app/src/main/java/io/stanwood/bitrise/ui/builds/ui/BuildsFragment.kt

```diff
@@ -44,4 +54,36 @@ class BuildsFragment : Fragment() {
             lifecycle.addObserver(viewModel)
             vm = viewModel
         }.root
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setHasOptionsMenu(true)
+        val currentActivity = activity
+        if (currentActivity is AppCompatActivity) {
+            toolbar.let {
+                currentActivity.setSupportActionBar(it)
+            }
+        }
+    }
+
+    override fun onCreateOptionsMenu(menu: Menu?, inflater: MenuInflater?) {
+        if (viewModel.isProvidedByGithub) {
+            inflater?.inflate(R.menu.builds, menu)
```

Comment:
I'd always inflate the option menu and just enable/disable the github item for consistency.